KarhooSDK/Api/Response/QuotePrice.swift

```diff
@@ -13,38 +13,30 @@ public struct QuotePrice: Codable {
     public let highPrice: Double
     public let lowPrice: Double
     public let currencyCode: String
-
-    // Server side price format
-    public let intHighPrice: Int
-    public let intLowPrice: Int
+    public let net: NetPrice
 
     public init(highPrice: Double = 0,
                 lowPrice: Double = 0,
                 currencyCode: String = "",
-                intLowPrice: Int = 0,
-                intHighPrice: Int = 0) {
+                net: NetPrice = NetPrice()) {
         self.highPrice = highPrice
         self.lowPrice = lowPrice
-        self.intHighPrice = intHighPrice
-        self.intLowPrice = intLowPrice
         self.currencyCode = currencyCode
+        self.net = net
     }
 
     enum CodingKeys: String, CodingKey {
         case highPrice = "high"
         case lowPrice = "low"
         case currencyCode = "currency_code"
+        case net = "net"
```

Comment:
Don't really need to set "net" here. Because the case itself is called net it will already know the object name it is looking for inside the JSON response. Shouldn't matter though, but just thought I would mention.